src/SbManager/Scripts/manager/homeController.js

```diff
@@ -1,9 +1,17 @@
 ﻿$app.controller('homeController', ['$scope', '$routeParams', function ($scope, $routeParams) {
     $scope.refresh = function () {
         $scope.model = null;
-        $.getJSON(window.applicationBasePath + "/api/v1/busmanager/", {}, function (d) {
-            $scope.model = d;
-            $scope.$digest();
+        $.ajax({
```

Comment:
I've actually changed all the calls to $http in https://github.com/jafin/SbManager/tree/Angular14 .  If that is of any use.